docs/best-practices-and-faq.html

```diff
@@ -300,9 +300,9 @@ <h4 id='what-39-s-the-purpose-of-code-iglistcollectionview-code' class='heading'
 <h4 id='how-can-i-manage-cell-selection-and-deselection' class='heading'>How can I manage cell selection and deselection?</h4>
 
 <p>See discussion at <a href="https://github.com/Instagram/IGListKit/issues/184">#184</a>.</p>
-<h4 id='i-have-a-em-huge-em-data-set-and-a-href-https-instagram-github-io-iglistkit-classes-iglistadapter-html-c-objc-cs-iglistadapter-im-performupdatesanimated-completion-code-peformupdatesanimated-completion-code-a-is-em-super-em-slow-what-do-i-do' class='heading'>I have a <em>huge</em> data set and <a href="https://instagram.github.io/IGListKit/Classes/IGListAdapter.html#/c:objc(cs)IGListAdapter(im)performUpdatesAnimated:completion:"><code>-peformUpdatesAnimated: completion:</code></a> is <em>super</em> slow. What do I do?</h4>
+<h4 id='i-have-a-em-huge-em-data-set-and-a-href-https-instagram-github-io-iglistkit-classes-iglistadapter-html-c-objc-cs-iglistadapter-im-performupdatesanimated-completion-code-performupdatesanimated-completion-code-a-is-em-super-em-slow-what-do-i-do' class='heading'>I have a <em>huge</em> data set and <a href="https://instagram.github.io/IGListKit/Classes/IGListAdapter.html#/c:objc(cs)IGListAdapter(im)performUpdatesAnimated:completion:"><code>-performUpdatesAnimated: completion:</code></a> is <em>super</em> slow. What do I do?</h4>
 
-<p>If you have multiple thousands of items and you cannot batch them in, you&rsquo;ll see performance issues with <code>-peformUpdatesAnimated: completion:</code>. The real bottle neck behind the scenes here is <code>UICollectionView</code> attempting to insert so many cells at once. Instead, call <a href="https://instagram.github.io/IGListKit/Classes/IGListAdapter.html#/c:objc(cs)IGListAdapter(im)reloadDataWithCompletion:"><code>-reloadDataWithCompletion:</code></a> when you first load data, which will be super fast. Behind the scenes, this method <em>does not</em> do any diffing and simply calls <code>-reloadData</code> on <code>UICollectionView</code>. For subsequent updates, you can then use <code>-peformUpdatesAnimated: completion:</code>.</p>
+<p>If you have multiple thousands of items and you cannot batch them in, you&rsquo;ll see performance issues with <code>-performUpdatesAnimated: completion:</code>. The real bottle neck behind the scenes here is <code>UICollectionView</code> attempting to insert so many cells at once. Instead, call <a href="https://instagram.github.io/IGListKit/Classes/IGListAdapter.html#/c:objc(cs)IGListAdapter(im)reloadDataWithCompletion:"><code>-reloadDataWithCompletion:</code></a> when you first load data, which will be super fast. Behind the scenes, this method <em>does not</em> do any diffing and simply calls <code>-reloadData</code> on <code>UICollectionView</code>. For subsequent updates, you can then use <code>-performUpdatesAnimated: completion:</code>.</p>
```

Comment:
we'll need to revert this and make the changes in `Guides/Best Practices and FAQ.md` instead 😄